### PR TITLE
QUEST_LV_0200 000087, 000101, 000104

### PR DESCRIPTION
**Pull request by BloodHeartless**
Simple changes

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -98,10 +98,10 @@ QUEST_LV_0200_20150317_000097	The sacrifice will continue unless someone stops t
 QUEST_LV_0200_20150317_000098	Joint Investigation Team Fray
 QUEST_LV_0200_20150317_000099	I'm Fray.{nl}Listening to the story, the situation sounds bad.{nl}If the information you and Scott researched is correct.
 QUEST_LV_0200_20150317_000100	{nl}Then, I will have to do something.[nl}Please defeat some Red Vubbes first. They have been bothering me lately.
-QUEST_LV_0200_20150317_000101	Please, I ask of you. .{nl}I get sick just looking at the red color.
+QUEST_LV_0200_20150317_000101	$Please, I ask to you. .{nl}I get sick just looking at the red color.
```
Comment (terrapin47) [inline QUEST_LV_0200.tsv:101]: "ask to you" is incorrect.  "ask of you" was correct.

Comment (BloodHeartless) [inline QUEST_LV_0200.tsv:101]: both is correct. But "to" is less confuse. This game is a game international  =)

Comment (ttgmichael) [inline QUEST_LV_0200.tsv:101]: @darkhershel Wrong sentence to use "ask to" in; "Ask to" is only correct when you have something in between: you can ask SOMETHING to someone. Or if the to is conjugating a verb; Ask to go to the restroom.
"Ask of" is used as a phrase on its own; Ask of you to do something. "May I ask a favour of you?"

The game is international in the sense that it uses American English, and I'm a native English speaker. xD

Comment (terrapin47) [inline QUEST_LV_0200.tsv:101]: No, "ask to you" is absolutely incorrect.

Comment (BloodHeartless) [inline QUEST_LV_0200.tsv:101]: yeah, my mistake.I'm sorry @ttgmichael @zainezed01

**Pull request closed without merging**